File: 00_AppJava/app/src/main/java/com/example/a00_appjava/MainActivity.java
package com.example.a00_appjava;

import android.os.Bundle;
import android.util.Log;
import android.view.View;
import android.widget.Button;

import androidx.appcompat.app.AppCompatActivity;

import java.util.concurrent.atomic.AtomicInteger;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Button boton1 = findViewById(R.id.boton1);
        boton1.setOnClickListener(view -> Log.i("MI_TAG", "Botón pulsado "));


        Button boton2 = findViewById(R.id.boton2);

        boton2.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Log.i("MI_TAG2", "Botón2 pulsado ");

            }
        });

    }
}

File: 00_AppKotlin/app/src/main/java/com/example/a00_appkotlin/MainActivity.kt
package com.example.a00_appkotlin

import android.os.Bundle
import android.util.Log
import android.view.View
import android.widget.Button
import androidx.activity.enableEdgeToEdge
import androidx.appcompat.app.AppCompatActivity
import androidx.core.view.ViewCompat
import androidx.core.view.WindowInsetsCompat

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val boton1: Button = findViewById(R.id.boton1)
        val boton2 = findViewById<Button>(R.id.boton2) // Hay que pasarle en el operador diamante el tipo de dato, porque estamos inicializando la variable con inferencia, sin declarar el tipo

        boton2.setOnClickListener {Log.i("tagg", "menasje")}





    }
}